ZenPacks/zenoss/Dashboard/browser/resources/js/defaultportlets.js

```diff
@@ -340,7 +340,9 @@
         applyConfig: function(config) {
             this.callParent([config]);
             if (this.rendered){
-                this.onRefresh();
+                if(config['baselocation'].match('\/zport\/dmd\/Locations\/')) {
+                    this.onRefresh();
+                }
```

Comment:
In the ticket, the issue is that `/Critical Space/` fails, but `/Critical Space/Contact Center` succeeds.  How does refreshing only when `/zport/dmd/Locations/` is found in the `baselocation` attribute fix the failure when `/Critical Space/` is selected?